packages/syft/src/syft/service/network/network_service.py

```diff
@@ -370,7 +347,8 @@ def get_peers_by_type(
     @service_method(
         path="network.delete_peer_by_id",
         name="delete_peer_by_id",
-        roles=DATA_OWNER_ROLE_LEVEL,
+        # roles=DATA_OWNER_ROLE_LEVEL,
+        roles=GUEST_ROLE_LEVEL,
```

Comment:
question: Shouldn't this be DATA OWNER ROLE LEVEL ?

Reply:
Reverted back to `DATA_OWNER_ROLE_LEVEL`